pepper-apis/docs/specification/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "pepper-api-spec",
-  "version": "1.25.0-71",
+  "version": "1.27.370",
```

The version value is a value from https://github.com/broadinstitute/ddp-study-server/pull/1345/files#r841690210,
which is formatted in order to be a valid [SemVer](https://semver.org/).

---

Small edit to get this a bit closer to the `dss-server` version (since we make use of the `PATCH` field)
```suggestion
  "version": "1.27.0-370",
```

---

Fixed in 2172e23b37204856fee31aa5a3b741d6f341bee8